lte/gateway/c/core/oai/tasks/grpc_service/AmfServiceImpl.cpp

```diff
@@ -102,10 +103,16 @@ Status AmfServiceImpl::SetSmfSessionContext(
   char ip_v6_str[INET6_ADDRSTRLEN] = {0};
 
   uint32_t ip_int = 0;
+  uint32_t i = 0;
+  traffic_flow_template_t* ul_tft = NULL;
+  traffic_flow_template_t* dl_tft = NULL;
+  int ul_count_packetfilters = 0;
+  int dl_count_packetfilters = 0;
   OAILOG_INFO(LOG_UTIL,
               "Received GRPC SetSmfSessionContext request from SMF\n");
 
   itti_n11_create_pdu_session_response_t itti_msg;
```

Comment:
No need of memset, itti_msg can be initialized as itti_n11_create_pdu_session_response_t itti_msg = {0};